openmicroscopy/openmicroscopy
# 32 bit and float (rebased onto dev_5_0)

This is the same as gh-2955 but rebased onto dev_5_0.

----

Add support for  32-bits images (pixels type)
 * Import the following images. 
    * test_image_good/ubm
    * from skyking/gatan/chip/
   * from skyking/gatan/qa-7642
 * Check that you can view the images.
 * Open the images in ImageJ or Fiji using BioFormats to validate.

Images imported w/o this PR can be found under 
trout/ port 14064 (latest), user-1 private-1 Project imported 18-08-14 to test 32 bit

Add support for floating point image
To test this PR

make sure that the unit tests under server/test/omeis.providers.re.utests are green
Import floating point images. a selection can be found under squig/team/will/float0-data.zip
View the image in OMERO, modify some values e.g. pixels window.
View the same image in ImageJ.
Note that the pyramid generation for floating point images (>4kx4k)will not work. This will not be addressed in this PR.
                
--rebased-from #2955 
--rebased-from #3033
